Assets/BossRoom/Scripts/Client/Game/Character/ClientInputSender.cs

```diff
@@ -97,7 +97,7 @@ private struct ActionRequest
         /// </summary>
         CharacterClass CharacterData => GameDataSource.Instance.CharacterDataByType[m_NetworkCharacter.CharacterType];
 
-        public override void NetworkStart()
+        public override void OnNetworkSpawn()
```

Comment:
as discussed, let's make sure there's a despawn that unregisters this if needed

Reply:
Done. Added an event to subscribe to (`HeroActionBar` and `HeroEmoteBar` do this) for when the local player's `ClientInputSender` is NetworkSpawned & NetworkDespawned.